src/logger.h

```diff
@@ -115,7 +114,6 @@ class Logger : public QObject
     QFile _file;
     QTextStream _fileStream;
     QStringList _registeredCategories;
-    QSettings _settings;
```

Comment:
Can we assign SettingsManager::self()->settings() to a member _settings, and use that everywhere for better readability?

Reply:
Yep, Will do that